### DISCUSSION TITLE
Contribution activity

### DISCUSSION THREAD

**Discussion by Juggernauff** [community/community#23603] [New to GitHub]
<p>Two years ago, at the university where I am studying, I was given tasks, the result of which I had to send to a shared repository. In this regard, the question is: How do I delete the activity (messages in the activity feed) that were two years ago?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<aside class="quote no-group" data-username="Juggernauff" data-post="1" data-topic="221174">
<div class="title">
<div class="quote-controls"></div>
 Juggernauff:</div>
<blockquote>
<p>How do I delete the activity (messages in the activity feed) that were two years ago?</p>
</blockquote>
</aside>
<p>are you referring to the Contribution activity? this one?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/b/eb7851dea35d8b60cb85e15aee378e5c0ad5ff67.png" data-download-href="/uploads/short-url/xB3ZlOoU6OkGWX0qvxuoi3jDht5.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078165-278688af-64ea-4a43-ad13-3dbaf4da58ef.png" alt="image" data-base62-sha1="xB3ZlOoU6OkGWX0qvxuoi3jDht5" width="690" height="391" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/b/eb7851dea35d8b60cb85e15aee378e5c0ad5ff67_2_690x391.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/b/eb7851dea35d8b60cb85e15aee378e5c0ad5ff67_2_1035x586.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/b/eb7851dea35d8b60cb85e15aee378e5c0ad5ff67_2_1380x782.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/b/eb7851dea35d8b60cb85e15aee378e5c0ad5ff67_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1389×789 48.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><p></p>
<p>check this out, you might have the same request,</p>
<aside class="quote" data-post="1" data-topic="880">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/is-there-a-way-to-remove-the-contribution-calendar-from-my-profile/880">Is there a way to remove the contribution calendar from my profile?</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    I’ve hunted around a bit in settings, in help, and here and haven’t found anything about this, but is there a way to hide or disable the contributions calendar (and, ideally, also the list) in my profile page? I’d like to simply have it not shown at all (or at least empty) if possible.
  </blockquote>
</aside>
